components/cache/cache_pools.rst

```diff
@@ -282,7 +282,7 @@ item in the cache immediately (it returns ``true`` if the item was saved or
 ``false`` if some error occurred)::
 
     // ...
-    $userFriends = $cache->get('user_'.$userId.'_friends');
+    $userFriends = $cache->getItem('user_'.$userId.'_friends');
```

Comment:
We now also need to call `get()` on the return value (the cache pool does not return the cached value, but a `CacheItemInterface` instance).